demos/insurance_agent/README.md

```diff
@@ -48,11 +48,13 @@ The system can perform a variety of tasks, such as answering insurance-related q
 Arch gateway publishes stats endpoint at http://localhost:19901/stats. In this demo we are using prometheus to pull stats from arch and we are using grafana to visalize the stats in dashboard. To see grafana dashboard follow instructions below,
 
 1. Start grafana and prometheus using following command
+
    ```yaml
    docker compose --profile monitoring up
    ```
-1. Navigate to http://localhost:3000/ to open grafana UI (use admin/grafana as credentials)
+1. Navigate to http://localhost:3000/ to op![Uploading Screenshot 2024-12-20 at 3.32.49 PM.png…]()
```

Comment:
This isn't rendering correctly. Please fix this image.

Reply:
Also, I think we should fix/remove the tests for the demos that are failing (e2e demo tests) in this branch. Much easier to clean up in a single go.